src/machine/machine_atmega328p.go

```diff
@@ -467,3 +472,71 @@ var SPI0 = SPI{
 	sdo:  PB3,
 	sdi:  PB4,
 	cs:   PB2}
+
+// Pin Change Interrupts
+var (
+	pinCallbacks [3][8]func(Pin)
+)
+
+func handlePCINTInterrupts(intr uint8, port *volatile.Register8) {
+	current := port.Get()
+	for i := uint8(0); i < 8; i++ {
+		if (current>>i)&0x01 == 0x01 && pinCallbacks[intr][i] != nil {
+			pinCallbacks[intr][i](Pin(i * (intr + 1)))
+		}
+	}
+}
+
+func handlePCINT0Interrupts(intr interrupt.Interrupt) {
+	handlePCINTInterrupts(avr.IRQ_PCINT0-avr.IRQ_PCINT0, avr.PINB)
+}
+
+func handlePCINT1Interrupts(intr interrupt.Interrupt) {
+	handlePCINTInterrupts(avr.IRQ_PCINT1-avr.IRQ_PCINT0, avr.PINC)
+}
+
+func handlePCINT2Interrupts(intr interrupt.Interrupt) {
+	handlePCINTInterrupts(avr.IRQ_PCINT2-avr.IRQ_PCINT0, avr.PIND)
+}
+
+type PinChange uint8
+
+// Pin change interrupt constants for SetInterrupt.
+const (
+	PinBoth PinChange = nil
```

Comment:
The standard name for this is `PinToggle` (because it triggers on every toggle between low and high). See https://tinygo.org/docs/reference/machine/.

(Also, did you test this code? I don't think you can assign `nil` to a uint8).

Reply:
I haven't tested it yet because `make release` is crashing my system at the moment. Just started yesterday. It's probably unrelated to _specifically_ to `make release`, because the first crash I had yesterday seemed to be related to Outlook. 
But after that, `make release` reliably crashes my computer. I probably need to reimage it, which I'm not looking forward too, but pushing up to GitHub ensures I don't lose this code.

I promise not to merge until I get that figured out and actually test it.